tools/vms-generator/utils/utils.go

```diff
@@ -472,7 +483,8 @@ func GetVMISRIOV() *v1.VirtualMachineInstance {
 	vm.Spec.Domain.Resources.Requests[k8sv1.ResourceMemory] = resource.MustParse("1024M")
 	vm.Spec.Networks = []v1.Network{*v1.DefaultPodNetwork(), {Name: "sriov-net", NetworkSource: v1.NetworkSource{Multus: &v1.MultusNetwork{NetworkName: "sriov/sriov-network"}}}}
 	initFedora(&vm.Spec)
-	addNoCloudDiskWitUserData(&vm.Spec, "#!/bin/bash\necho \"fedora\" |passwd fedora --stdin\ndhclient eth1\n")
+	networkData := "version: 2\nethernets:\n  eth1:\n    dhcp4: true\n"
```

Comment:
nit pick: maybe export to const as you have the same `networkData` also on line 512:
https://github.com/kubevirt/kubevirt/blob/19ace22a6af331aaedc2ecb06949958af5802285/tools/vms-generator/utils/utils.go#L512

but it is in different function, and it is only two times. So I don't know.

Reply:
I can do this, but, I wonder if it makes sense in this PR's scope; check my answer to @ormergi 's comment: https://github.com/kubevirt/kubevirt/pull/6686#discussion_r742627405

Reply:
I actually went ahead and added this ... It's in a separate commit, so we can easily get rid of it if we don't want to have it.

Reply:
Thank you